internal/pkg/service/stream/components.go

```diff
@@ -27,26 +29,62 @@ type Components []Component
 
 type Component string
 
-func StartComponents(ctx context.Context, d dependencies.ServiceScope, cfg config.Config, components ...Component) error {
+func StartComponents(ctx context.Context, d dependencies.ServiceScope, cfg config.Config, components ...Component) (err error) {
 	componentsMap := make(map[Component]bool)
 	for _, c := range components {
 		componentsMap[c] = true
 	}
 
+	// Common distribution scope
+	var distScp commonDeps.DistributionScope
```

Comment:
`distScp` -> `distScope`, Combination of `dist` and `scp` is very weird as,  Scp is command for secure copy files, and I was unsure why we use that.

Reply:
Rename it however you want :slightly_smiling_face:  , but please in the whole repository.